pkg/orchestration/wiring/k8scompute/plugin.go

```diff
@@ -334,9 +335,9 @@ func (p *WiringPlugin) WireUp(resource *orch_v1.StateResource, context *wiringpl
 	podSpec := buildPodSpec(containers, serviceAccountNameRef.Ref(), affinity)
 
 	// Add pod disruption budget
-	pdbSpec := buildPodDisruptionBudgetSpec(labelMap, spec.Scaling.MinReplicas)
+	pdbSpec, pdbNamePostfix := buildPodDisruptionBudgetSpec(labelMap, spec.Scaling.MinReplicas)
 	pdb := smith_v1.Resource{
-		Name: wiringutil.ResourceNameWithPostfix(resource.Name, pdbPostfix),
+		Name: wiringutil.ResourceNameWithPostfix(resource.Name+"--"+voyager.ResourceName(pdbNamePostfix), pdbPostfix),
```

Comment:
I didn't add you as a reviewer, but do you object to this @ash2k ?

Reply:
I think it's the right way to do it, but we can do it in a cleaner way. I'll send a PR with small refactoring of helper functions and then you can use the better version of it.

Reply:
https://github.com/atlassian/voyager/pull/184

Reply:
After this is merged, you'll be able to do this: `ResourceName(resource.Name, pdbPostfix, pdbNamePostfix)`

Reply:
merged

Reply:
Fixed.